crates/sickle/src/parser.rs

```diff
@@ -208,4 +224,73 @@ database =
         let map = parse_to_map(input).unwrap();
         assert!(map.contains_key("database"));
     }
+
+    #[test]
+    fn test_unindented_line_not_continuation() {
+        // Per CCL spec: "Lines indented more than previous = part of that value"
+        // This means unindented lines should NOT be continuations
+        let input = r#"descriptions = First line
+second line
+descriptions = Another item"#;
```

Comment:
This is supposed to be two entries, [descriptions, First line] and [second line\ndescriptions, Another item].